Mon Jun 26 13:55:09 CEST 2006  alex_extreme <alex.extreme2@gmail.com>
  * libfwxconfig: replace imps2 with imps/2
diff -rN -u old-frugalwareutils/libfwxconfig/libfwxconfig.c new-frugalwareutils/libfwxconfig/libfwxconfig.c
--- old-frugalwareutils/libfwxconfig/libfwxconfig.c	2006-06-26 22:54:41.000000000 +0200
+++ new-frugalwareutils/libfwxconfig/libfwxconfig.c	2006-06-26 22:54:41.000000000 +0200
@@ -129,7 +129,7 @@
 		}
 		if(_fwx_reg_match(line, "Identifier.*Mouse"))
 		{
-			_fwx_print_mouse_identifier(nfp, 0, "/dev/psaux", "imps2");
+			_fwx_print_mouse_identifier(nfp, 0, "/dev/psaux", "imps/2");
 			_fwx_print_mouse_identifier(nfp, 1, "/dev/tts/0", NULL);
 			fprintf(nfp, "Identifier  \"Mouse3\"\n");
 			line[0]='\0';

